instrumentation/opentelemetry-instrumentation-urllib/src/opentelemetry/instrumentation/urllib/__init__.py

```diff
@@ -297,17 +297,17 @@ def _create_client_histograms(meter) -> Dict[str, Histogram]:
         MetricInstruments.HTTP_CLIENT_DURATION: meter.create_histogram(
             name=MetricInstruments.HTTP_CLIENT_DURATION,
             unit="ms",
-            description="measures the duration outbound HTTP requests",
+            description="Measures the duration of outbound HTTP requests.",
         ),
         MetricInstruments.HTTP_CLIENT_REQUEST_SIZE: meter.create_histogram(
```

Comment:
I think this has been [renamed](https://github.com/open-telemetry/semantic-conventions/commit/9db3c5fdd448ad100c090191fdac0f067f9d1cf4#diff-08cdc1cb5ff57be3c20e0899dae7b45a13c834ea2dad6af35e86218e0e52b8b8R20) to [`request.body.size`](https://github.com/open-telemetry/semantic-conventions/blame/main/model/metrics/http.yaml#L170).

Reply:
That does seem separate from changing the descriptions. Should we do that in another PR?

Reply:
This would be part of the new semantic convention migration and I don't think that should be part of this pr

Reply:
Would you like me to change the metric names to match the new semconv in this PR, or will that be done in a separate PR?